solutions/secure-cross-regional-bucket/variables.tf

```diff
@@ -1,6 +1,6 @@
 variable "ibmcloud_api_key" {
   type        = string
-  description = "The IBM Cloud platform API key to deploy IAM-enabled resources."
+  description = "The IBM Cloud API key to deploy IAM-enabled resources."
```

Comment:
Hmm, not sure it should say "IAM-enabled resources" because the apikey is also used to create account resources (such a s2s auth policies)

Reply:
That was in there before. I removed `IAM-enabled` and made it generic. 

Removed it from the variable from all `solutions/`